bin/provision-instances

```diff
@@ -0,0 +1,227 @@
+#!/usr/bin/python3
+#
```

Comment:
This is an ad-hoc script - the other scripts don't have a standard looking header. E.g. https://github.com/open-craft/opencraft/blob/master/bin/bootstrap.

Is there a special reason this one should have it, or are the other scripts just missing it as well?